package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "presto-js-client",
+  "version": "0.0.0",
+  "license": "MIT",
+  "scripts": {
+    "build": "nx build",
```

Comment:
I am not familiar with `Nx`, it seems useful and good to learn new stuffs. However, when I run `npm run build`, it stucks in the `nx build` loop.

Reply:
would it be `nx build presto-client`?

Reply:
`nx build` is just the alias so u can pass arts to it. The real command would be `npm run build presto-client`

Reply:
thanks for the explanation. I guess having a `build:presto-client: nx build presto-client` would be handy here since it's the package.json in the root of this repo. I saw the `README.md` under `presto-client` just now.

Reply:
Yeap we still have to update some of the readme files

Reply:
I added the "build:presto-client" script as well